### PR TITLE
Fix some styling issues on navbar dropdown. Also moves styling into c…

### PR DESCRIPTION
**Pull request by arwagner**
…ustom classes

### CODE REVIEW

File: front-end/src/components/Navigation/Navbar.scss [hunk 1]
```diff
@@ -8,46 +8,70 @@
...
     padding-bottom: 0;
     margin-right: 20px;
   }
 
+  .text {
+    color: white;
+  }
+
+  .icon {
+    color: white;
```
Comment (kdyer) [inline front-end/src/components/Navigation/Navbar.scss]: https://github.com/FearlessSolutions/sba-eli/blob/master/front-end/src/assets/styles/_colors.scss#L16

File: front-end/src/components/Navigation/Navbar.scss [hunk 2]
```diff
@@ -8,46 +8,70 @@
...
-
-    .learning-path-item:hover {
-      background-color: $light-blue !important;
-
-      .text {
-        color: white;
-      }
+    
+    .menu {
+      background-color: $grey1;
```
Comment (kdyer) [inline front-end/src/components/Navigation/Navbar.scss]: https://github.com/FearlessSolutions/sba-eli/blob/master/front-end/src/assets/styles/_colors.scss#L12

File: front-end/src/components/Navigation/Navbar.scss [hunk 3]
```diff
@@ -8,46 +8,70 @@
...
+
+    .learning-path-item {
+      background-color: $grey1 !important;
+      margin: 20px;
     }
 
-    .topic-summary {
-      margin-left: 15px;
+    .learning-path-item:hover {
+      background-color: $light-blue !important;
```
Comment (kdyer) [inline front-end/src/components/Navigation/Navbar.scss]: https://github.com/FearlessSolutions/sba-eli/blob/master/front-end/src/assets/styles/_colors.scss#L2

**Comment by kdyer** [comment]
Should use the base color variables defined in the color css file. Also if we have to use !important, the selector is probably not specific enough.

**Comment by arwagner** [comment]
Fixed the duplicated colors. Per our conversation, I don't think the !important can be gotten around this time, unfortunately.

**Pull request merged**